packages/core/i18n/nls.json

```diff
@@ -390,6 +390,9 @@
         "update-version-version-error": "Failed to install version {0} of {1}."
       }
     },
+    "vsx-extension": {
+      "averageRating": "Average rating: {0} out of 5"
+    },
```

Comment:
Can you please reset this change and rebase onto latest `master`? That should then automatically create the appropriate translation entries. You can then change the `zh-cn` translation if you don't find it fitting.

Reply:
@msujew Thank you, but I have a confusion. Do you mean that I can divide it into two steps to do localization?

In the first step, I only need to use `nls.localize('theia/vsx-extension/averageRating', 'Average rating: {0} out of 5', averageRatingFormatter(averageRating)` in the code. CI will automatically generate the corresponding entries in' nls.json' and' nls-zh-cn.json '. 

In the second step, if the default Chinese translation is not very suitable, then I modify the Chinese translation in' nls-zh-cn.json?